docs/reference/superjson.mdx

```diff
@@ -0,0 +1,294 @@
+---
+slug: /reference/superjson
+---
+
+# Super.json Reference
+
+Super.json is the main Superface configuration file. It contains the configuration about profiles, maps and providers used.
+
+## Anatomy
+
+At the top level, super.json file consists of two sections: [profiles](#profiles) and [providers](#providers). the main purpose profile section is to set up profiles which are to be used, their versions and/or file locations, and their default values. The purpose of provider section is to set up providers that are to be used and their credentials.
```

Comment:
```suggestion
At the top level, super.json file consists of two sections: [profiles](#profiles) and [providers](#providers). Profiles section contains list of installed profiles, their versions and/or file locations, and default values. Providers section contains list of installed providers and their credentials.
```
I tried to write it in simple English. Not sure about the term `used`. I would use `installed`.

Reply:
Working further from Jan's suggestion to add articles:
```suggestion
At the top level, the `super.json` file consists of two sections: [profiles](#profiles) and [providers](#providers). The profiles section contains a list of installed profiles, their versions or file locations, and default values. The providers section contains a list of installed providers and their credentials.
```